### DISCUSSION TITLE
Move to Beta channel

### DISCUSSION THREAD

**Discussion by rvaldomir** [sebanc/brunch#1022] [Q&A]
Hi,

If you are using the stable channel for Chrome OS image. How to move to Beta? 

I tried changing the channel in the updates configuration additional details, but it never pulls for the beta version.

Is there any other way to do this?

**Comment by formire** [comment]
did you enable the option for auto-updates in GRUB?

**Reply by rvaldomir** [reply]
mmm nop. should i?

**Reply by formire** [reply]
That's what I did to move to the dev channel

**Reply by rvaldomir** [reply]
worked thanks!

**Comment by gjlamb** [comment]
For some reason I couldn’t get the grub config updated for enable_updates via crosh shell or editing directly in grub boot loader.  Using brunch-toolkit to add options seemed to work enabling updates so I could change to beta channel
